File: CabBooking/Model/Location.cs
namespace CabBooking.Model
{
    class Location
    {
        int XCordinate;
        int YCordinate;
        public Location(int xCordinate, int yCordinate)
        {
            this.XCordinate = xCordinate;
            this.YCordinate = yCordinate;
        }
        public string GetDistance(Location destination)
        {
            return Math.Sqrt(
            Math.Pow(this.XCordinate - destination.XCordinate, 2)
            +
            Math.Pow(this.YCordinate - destination.YCordinate, 2)
            ).ToString();
        }
    }
}

File: CabBooking/Cab Booking Java/src/main/java/org/example/model/Location.java
package org.example.model;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public class Location {

    private final Double xCoordinate;
    private final Double yCoordinate;

    public Double computeDistance(Location location) {
        return Math.sqrt(
                Math.pow(location.xCoordinate - this.xCoordinate, 2) +
                        Math.pow(location.yCoordinate - this.yCoordinate, 2)
        );
    }
}
